docs/modules/examples/pages/mmr.adoc

```diff
@@ -2,21 +2,21 @@
 
 image::https://colab.research.google.com/assets/colab-badge.svg[align="left",link="https://colab.research.google.com/github/CassioML/cassio-website/blob/main/docs/frameworks/langchain/.colab/colab_qa-maximal-marginal-relevance.ipynb"]
 
-This page demonstrates using RAGStack and an Astra vector database to perform vector search with the *Maximal Marginal Relevance (MMR)* algorithm.
+This page demonstrates using RAGStack and an {db-vector} database to perform vector search with the *Maximal Marginal Relevance (MMR)* algorithm.
 
 Instead of selecting the top _k_ stored documents most relevant to the provided query, MMR first identifies a larger pool of relevant results, and then retrieves top _k_ from this pool. MMR algorithms return results with more diverse information.
 
 == Prerequisites
 
-. You will need a vector-enabled Astra database.
+. You will need an {db-vector} database.
 +
 .. Create an https://docs.datastax.com/en/astra-serverless/docs/getting-started/create-db-choices.html[Astra
 vector database].
 +
 .. Within your database, create an https://docs.datastax.com/en/astra-serverless/docs/manage/org/manage-tokens.html[Astra
 DB Access Token] with Database Administrator permissions.
 +
-.. Copy your Astra DB Endpoint from the Astra database.
+.. Copy your Astra DB Endpoint from the Astra DB database.
```

Comment:
Should we specify Astra DB Serverless here?

Reply:
Yes. Done across multiple files. Thanks!